frontend/src/lib/components/flows/propPicker/PropPickerWrapper.svelte

```diff
@@ -43,23 +43,19 @@
 	})
 </script>
 
-<HSplitPane leftPaneSize="50%" rightPaneSize="50%" minLeftPaneSize="20%" minRightPaneSize="20%">
-	<left slot="left" class="relative">
-		<div class="overflow-auto h-full p-4">
-			<slot />
-		</div>
-	</left>
-	<right slot="right">
-		<div class="overflow-auto h-full">
-			<PropPicker
-				{displayContext}
-				{pickableProperties}
-				on:select={({ detail }) => {
-					dispatch('select', detail)
-					$propPickerConfig?.onSelect(detail)
-					propPickerConfig.set(undefined)
-				}}
-			/>
-		</div>
-	</right>
-</HSplitPane>
+<Splitpanes>
+	<Pane minSize={20} class="relative p-4 !duration-[0ms]">
```

Comment:
why would we need to use `!duration-[0ms]`?

Reply:
It was sliding in from the side everytime the component was mounted, which was awkward looking.

Reply:
I think the sliding is an issue linked to the fact that both Parent and child wrapper are not taking as much height as possible. In the previous iteration, I've fixed it by wrapping everything in a div that uses: "flex flex-1" to grow